e2e/specs/drug-orders.spec.ts

```diff
@@ -1,228 +1,283 @@
-    const dataRow = medicationsPage.medicationsTable().locator('tbody > tr');
-
-    await expect(headerRow.nth(0)).toContainText(/start date/i);
-    await expect(headerRow.nth(0)).toContainText(/details/i);
-    await expect(dataRow.nth(0)).toContainText(/aspirin 81mg/i);
-    await expect(dataRow.nth(0)).not.toContainText(/1 tablet/i);
-    await expect(dataRow.nth(0)).toContainText(/2 tablet/i);
-    await expect(dataRow.nth(0)).not.toContainText(/oral/i);
-    await expect(dataRow.nth(0)).toContainText(/inhalation/i);
-    await expect(dataRow.nth(0)).not.toContainText(/once daily/i);
-    await expect(dataRow.nth(0)).toContainText(/twice daily/i);
-    await expect(dataRow.nth(0)).not.toContainText(/3 days/i);
-    await expect(dataRow.nth(0)).toContainText(/5 days/i);
-    await expect(dataRow.nth(0)).not.toContainText(/indication headache/i);
-    await expect(dataRow.nth(0)).toContainText(/indication hypertension/i);
-  });
-
-  await test.step('When I click the overflow menu in the table row with the updated medication', async () => {
-    await page
-      .getByRole('button', { name: /options/i })
-      .nth(0)
-      .click();
-  });
-
-  await test.step('And I click on the "Discontinue" button', async () => {
-    await page.getByRole('menuitem', { name: /discontinue/i }).click();
-  });
-
-  await test.step('Then the order status should be changed to "Discontinue"', async () => {
-    await expect(orderBasket.getByText(/discontinue/i)).toBeVisible();
-  });
-
-  await test.step('And I click on the "Sign and close" button', async () => {
-    await page.getByRole('button', { name: /sign and close/i }).click();
-  });
-
-  await test.step('Then I should see a success notification', async () => {
-    await expect(page.getByText(/discontinued aspirin 81mg/i)).toBeVisible();
-  });
-
-  await test.step('And the medications table should be empty', async () => {
-    await expect(page.getByText(/there are no active medications to display for this patient/i)).toBeVisible();
+test.describe.serial('Drug Order Tests', () => {
+  test('Record a drug order', async ({ page }) => {
+    const orderBasket = page.locator('[data-extension-slot-name="order-basket-slot"]');
+
+    await test.step('When I visit the medications page', async () => {
+      await page.goto(url + `/spa/patient/${patient.uuid}/chart/Medications`);
+    });
+
+    await test.step('And I click to Add a drug order ', async () => {
+      await page.getByRole('button', { name: 'Add', exact: true }).click();
+    });
+
+    await test.step('And I search for "Aspirin" in the search bar', async () => {
+      await page.getByRole('searchbox', { name: /search for a drug or orderset/i }).fill('aspirin');
+    });
+
+    await test.step('And I add "Aspirin 81mg" to the basket', async () => {
+      await page
+        .getByRole('listitem')
+        .filter({ hasText: /aspirin 325mg — 325mg — tablet/i })
```

Comment:
Can't _just_ change the description. The description should match what we're looking for, so if this is meant to be Aspirin 81mg, then we should find that, etc.

Reply:
It looks like there was a mismatch between the test assertions and actions and the test description, which is what @gitcliff fixed here.